src/libs/actions/Session/index.js

```diff
@@ -165,45 +165,6 @@ function resendValidateCode(login = credentials.login) {
     API.write('RequestNewValidateCode', {email: login}, {optimisticData, successData, failureData});
 }
 
-/**
- * Request a new validate / magic code for user to sign in automatically with the link
- *
- * @param {String} [login]
- */
-function resendLinkWithValidateCode(login = credentials.login) {
```

Comment:
@cristipaval just to confirm this function gets added earlier and used, but now seems we are not using it. So I removed it in this PR. Can you plz cross-check once?

Reply:
Yes, I just checked and it looks like it is not used anymore.